src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -41,11 +41,6 @@
       References capability, but everything else we want back.
     -->
     <ProjectCapability Include="
-                          AssemblyReferences;
```

Comment:
I think @davkean asked this question elsewhere but this means that every project needs the SDK package to work reasonably with this project system. That seems too restrictive.  I don't understand what moving this to the SDK gives us.